asyn/drvAsynSerial/drvAsynSerialPortWin32.c

```diff
@@ -565,6 +578,13 @@ static asynStatus writeIt(void *drvPvt, asynUser *pasynUser,
         }
     }
     if (timerStarted) epicsTimerCancel(tty->timer);
+	
+	/* Sleep after sending bytes if requested */
+	if (tty->sleep_len > 0) {
+		FlushFileBuffers(tty->commHandle); /* ensure all data transmitted prior to sleep */
+		Sleep(tty->sleep_len);
```

Comment:
I think it would be worth moving the flush to before this test so a flush is done even if no delay is specified

Reply:
Ah, just noticed there is a flush in the next section, so it will get done. Hmm ... you get two flushes if you specify both delay and duration. I do wonder about moving the sleep into the if branch below?